### ISSUE TITLE
Nick Gorbunov (Winners League)

### ISSUE THREAD

**Issue by PeterHKang** [jonwalch/whiplash#429]
TL;DR: He likes our extension a lot. Think it's a good lead to pursue if and when we exhaust FACEIT/ESL/NBA. He wants more of a B2B solution that keeps traffic on their channels, but if we built him that he'd be super into it. (maybe even pay?)

Last season got 3K viewers. Next season starting Late October.
Staying away from embeds as much as possible. Twitch doesn't like embeds and hasn't granted them partnership
Takes Twitch 3 weeks to respond to them with a non-answer

Very interested in 3rd party apps that help with viewer engagement and retention.
Interested in companion apps (stats tracking, media, etc.)
Whiplash -> "Very promising product in the making"

How he sees Whiplash being applied:
- A Winner's league specific leader board that spans over multiple matches. Per each section of a season: group stage, Qualifiers, etc.
- Partnered with Toshiba last year. Working on other brands like this now through October. Would want to connect Whipcash leaderboard to sponsored prizes
- *Seems like he was concerned with getting returning viewers more than increasing session duration*

Examples in market he mentioned:
- Voting on the MVP
- Voting on who the top fragger is
- HLTV fantasy league leaderboards and prize giveaways
